packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,166 @@ export const pageQuery = graphql`
+
+<LivePreview scope={{Card, Paragraph, Heading, Button, PlusIcon}} language="jsx">
   {`<Card padding="space200">
-  <Heading as="h2">With some padding</Heading>
-  <Text as="p">Body</Text>
-</Card>`}
+  <div style={{textAlign: 'center'}}>
+    <PlusIcon decorative title="There just wasn't a details icon to use"/>
+    <Heading as="h2">Let's verify your integration</Heading>
+    <Paragraph>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua.
+    </Paragraph>
+    <Button variant="submit"> Test my integration </Button>
+  </div>
+</Card>;`}
 </LivePreview>
+{/* 
+We didn't have an icon or grid/layout components so we'll save this example for later.
+#### Card with Custom Layout
 
-
-<LivePreview scope={{Card, Text, Heading}} language="jsx">
-  {`<Card aria-busy aria-atomic data-qahook="ZeCard!" >
-  <Heading as="h2">Prop Passthrough</Heading>
-  <Text as="p">Body</Text>
+<LivePreview scope={{Card, Paragraph, Heading, Button}} language="jsx">
+  {`<Card>
+  <Heading as="h2">Heading Two</Heading>
+  <Paragraph>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam dui tellus, tristique in tincidunt cursus, efficitur
+    at felis. Phasellus imperdiet, lorem et commodo egestas, arcu.
+  </Paragraph>  
 </Card>`}
-</LivePreview>
+</LivePreview> */}
+
+## Composing a Card
+### When to use a Card
+Cards are a great tool for relating a concise amount of related information together in one object, much like a business card or baseball card in real life. 
+
+For example, you might use a single Card as a summary for a feature whose call to action encourages a user to dive deeper. Alternatively, multiple Cards on a single page with hierarchy of equal weights could be used to highlight multiple choices or paths a user can take. 
+
+Ultimately, Card is a very flexible component that can fit a variety of needs. If you feel that you have a unique use case or are limited by the Card implementation, please visit Design System Office Hours to see how we can help address your needs. 
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Have a clear call to action when including Buttons. Avoid including more than one primary and one secondary button."
+    preview={false}
+  >
+    <Img fluid={props.data.primaryButtonDo.childImageSharp.fluid} />
+  </Do>
+  <Dont title="Don't" body="Don’t put multiple primary actions in a Card" preview={false} >
+    <Img fluid={props.data.primaryButtonDont.childImageSharp.fluid} />
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use a Heading in your Card that appropriately reflects the information architecture of the page." preview={false} >
+    <Img fluid={props.data.headingInCardDo.childImageSharp.fluid} />
+  </Do>
+  <Dont title="Don't" body={<>Don’t use Cards to highlight multiple primary actions on a single page. <br /><br />
+    Don’t use multiple Heading components in one Card</>} preview={false} >
```

Comment:
Seems like the second line is here by mistake since it's repeated (ish) in the next "Don't"